ts/services/audioRecorder.ts

```diff
@@ -40,7 +54,15 @@ export class RecorderClass {
     }
   }
 
-  async start(): Promise<void> {
+  async start(): Promise<boolean> {
+    const microphonePermission = await this.requestMicrophonePermissions();
+    if (!microphonePermission) {
+      log.info(
+        'Microphone permission was denied, new audio recording not allowed.'
```

Comment:
Would be nice to prefix this log with `Recorder/start` so we can find all recorder type logs easier in debug logs.